composeApp/src/commonMain/kotlin/world/respect/app/view/ReportScreen.kt

```diff
@@ -0,0 +1,26 @@
+package world.respect.app.view
+
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.collectAsState
+import androidx.compose.runtime.getValue
+import androidx.compose.material3.Text
+import org.kodein.di.compose.localDI
+import world.respect.app.viewmodel.ReportScreenViewModel
+
+@Composable
+fun ReportScreen() {
+    val di = localDI()
+    val viewModel = ReportScreenViewModel(di)
```

Comment:
Wrong way to handle a view model:  this is going to make a new view model every single time the composable function is run. Please use view models as per the official docs I linked to in the main pull request conversation.